src/reducers/controls.ts

```diff
@@ -512,15 +512,8 @@ const Controls = (state: ControlsState = getDefaultControlsState(), action): Con
     case types.TOGGLE_MEASUREMENTS_THRESHOLD: // fallthrough
     case types.APPLY_MEASUREMENTS_FILTER:
       return {...state, ...action.controls};
-    /**
-     * Currently the CHANGE_ZOOM action (entropy panel zoom changed) does not
-     * update the zoomMin/zoomMax, and as such they only represent the initially
-     * requested zoom range. The following commented out code will keep the
-     * state in sync, but corresponding changes will be  required to the entropy
-     * code.
-     */
-    // case types.CHANGE_ZOOM: // this is the entropy panel zoom
-    //   return {...state, zoomMin: action.zoomc[0], zoomMax: action.zoomc[1]};
+    case types.CHANGE_ZOOM: // this is the entropy panel zoom
+      return {...state, zoomMin: action.zoomc[0], zoomMax: action.zoomc[1]};
```

Comment:
I remember disabling this as it caused bugs, or maybe unnecessary renders, but I can't remember the specifics and 2a063a706b82e8b8938bdb6ceb9f5a3bde85b11c doesn't spell them out either. I'll do some more testing when the PR's ready to go!

Reply:
> or maybe unnecessary renders

This is it I think. With these changes we now trigger two calls to `update` rather than one. The first has no zoom information, the second sets min/max to the overall zoom bounds. (Sometimes three calls, but my suggested changes in #1947 will limit it to two calls.)

I don't think this is blocking, although would be worth spending a small amount of time to see if the logic can be improved so there's only one `update` call.